apps/testing-javascript/src/path-to-purchase/pricing.tsx

```diff
@@ -102,7 +123,10 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
   const appliedMerchantCoupon = formattedPrice?.appliedMerchantCoupon
 
   // DON'T DELETE IT!!!
-  const pppCoupon = getFirstPPPCoupon(formattedPrice?.availableCoupons)
+  const pppCoupon = getFirstPPPCoupon(
+    formattedPrice?.availableCoupons,
+    merchantCoupon,
+  )
```

Comment:
I'm having this function isolate an available PPP Coupon from the available coupons and merchant coupon so that we don't have to do `pppCoupon | merchantCoupon` throughout the code below.